internal/commands/document.go

```diff
@@ -38,26 +38,32 @@ func newDocCommand() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringP("output", "o", "policies.md", "Output location (including filename) for the policy documentation")
+	cmd.Flags().StringP("output", "o", "", "Output location (including filename) for the policy documentation")
 
 	return &cmd
 }
 
 func runDocCommand(path string) error {
-	policyDocumentation, err := getPolicyDocumentation(path)
+	outputDirectory := filepath.Dir(viper.GetString("output"))
+	policyDocumentation, err := getPolicyDocumentation(path, outputDirectory)
 	if err != nil {
 		return fmt.Errorf("get policy documentation: %w", err)
 	}
 
-	err = ioutil.WriteFile(viper.GetString("output"), []byte(policyDocumentation), os.ModePerm)
+	outputPath := viper.GetString("output")
+	if outputPath == "" {
+		outputPath = "policies.md"
```

Comment:
Why are we using a conditional here rather than setting a default value in the flag?

Reply:
I had a reason for this initially... but I don't recall why. Shouldn't be needed. I'll fix this.

Reply:
Fixed!